pkg/rrd/rrd.go

```diff
@@ -203,12 +195,11 @@ func (r *RRD) SafeUpdate(timestamp time.Time, values []float64) error {
 // This method adds multiple graphs (e.g., hourly, daily, weekly, etc.) to the RRD.
 func (r *RRD) initGraphs() {
 	// Define the list of time lengths and consolidation functions for each graph.
-	timeLengthsMax := []string{"1h", "4h", "8h"}                 // Only use "MAX" for these time lengths.
-	timeLengthsAverage := []string{"1d", "4d", "1w", "1m", "1y"} // Only use "AVERAGE" for these time lengths.
+	timeLengthsMax := []string{"15m", "4h", "8h", "1d", "4d", "1w"} // Only use "MAX" for these time lengths.
```

Comment:
as @sarcasticadmin said. SCaLE is only 1 week. Eventually these should be configurable at run time or via config.